.travis.yml

```diff
@@ -1,16 +0,0 @@
-services:
-  - docker
-
-before_install:
-  - docker pull sifive/wit:0.10.1
-
-install:
-  - docker run -d -v `pwd`:`pwd` -w `pwd` -i -t --name wit-host sifive/wit:0.10.1 tail -f /dev/null
-  - docker exec -it wit-host bash -c "git config --global url."https://github.com/".insteadOf 'git@github.com:'"
-  - docker exec -it wit-host bash -c "wit init workspace -a $PWD -a git@github.com:sifive/environment-blockci-sifive.git"
-  - docker build --file=soc-freedom-sifive.Dockerfile --tag=sim .
```

Comment:
I totally forgot that I had asked Leyton to to create that Dockerfile. It was a not-quite-fleshed-out attempt at being able to extend the Docker image from env-blockci-sifive. I think it was only ever used to set up CI for this repo, so you can delete this now that we're using env-blockci-sifive proper.

Reply:
I do not know how to delete docker images.

Reply:
It's not an image, but a Dockerfile. You can just delete the file in this repo named `soc-freedom-sifive.Dockerfile`.

Reply:
Ah.